lib/packager/executor.rb

```diff
@@ -52,8 +52,11 @@ def create_package_for(item)
     end
 
     def execute_command(cmd)
-      return if dryrun
-
+      if dryrun
+        puts 'Dry run! This has not run or changed anything'
+        puts "#{cmd.to_system.join(' ')}"
+      end
```

Comment:
This needs to have a `return` in this block in order to avoid the invocation of the command.

Reply:
Good catch!